uint8 Type_Images=0
uint8 Type_Switch=1
uint8 Type_Tristate=2
uint8 Type_AnalogValue=3
uint8 FeatureType
string UUID
string Name
string Description
